src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -122,6 +122,18 @@ public string HubName
         /// </remarks>
         public bool? LocalRpcEndpointEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets the maximum number of entity operations that are processed as a single batch.
+        /// </summary>
+        /// <remarks>
+        /// Reducing this number can help to avoid timeouts on consumption plans. If set to 1, batching is disabled, and each operation
+        /// message executes and is billed as a separate function invocation.
+        /// </remarks>
+        /// <value>
+        /// A positive integer configured by the host.
+        /// </value>
+        public int? MaxEntityOperationBatchSize { get; set; } = null;
```

Comment:
Can we also create PRs to update the documentation for this (both host.json reference and the entities documentation)?